templates/common/github_provider.tmpl

```diff
@@ -0,0 +1,10 @@
+{{define "github_provider"}}
+
+provider "github" {
```

Comment:
should we start adding a version requirement on these provider templates? or allowing for one? I've ran into some kind of insidious bugs because I under-specified the bless provider version

Reply:
good point